package.json

```diff
@@ -58,7 +62,9 @@
     "ts-morph": "22.x",
     "typescript": "5.x"
   },
+  "packageManager": "pnpm@9.5.0",
```

Comment:
Sorry for the late comment, but the latest version is now 9.6.0, so please update it.

Reply:
You're welcome, Updated.

Reply:
Thanks!